infrastructure/main.tf

```diff
@@ -119,13 +119,13 @@ module "tribunals-case-api" {
 
     IDAM_S2S_AUTH_TOTP_SECRET="${data.vault_generic_secret.cmc_s2s_secret.data["value"]}"
     IDAM_S2S_AUTH_MICROSERVICE="${var.idam_s2s_auth_microservice}"
-    IDAM_S2S_AUTH_URL="${data.vault_generic_secret.idam_s2s_api.data["value"]}"
+    IDAM_S2S_AUTH_URL="${var.env == "prod" ? data.vault_generic_secret.idam_s2s_api.data["value"] : local.s2sCnpUrl}"
```

Comment:
Just a quick question why do we need this condition.For prod we are retrieving from vault?

Reply:
Not needed anymore, wanted to test S2S in AAT first before moving same in prod, but as case of urgency, and module compatibility issues this will be removed in next commit :)